index.html

```diff
@@ -32,96 +32,100 @@ <h3> I'm a software developer </h3>
-  </form>
-  <form class="segment"> 
-    <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
-    <button type="button" class="see-project"> See Project </button>
-  </form> <form class="segment"> 
-    <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
-    <button type="button" class="see-project"> See Project </button>
-  </form>
-  <form class="segment"> 
-    <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
-    <button type="button" class="see-project"> See Project </button>
-  </form>
-  <form class="segment"> 
+    <div class="see-project-button">
+      <button type="submit">See Project</button>
+    </div>
+    </div>
+  </section>
+  <section class="box">
+    <form>
     <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
+      <h1> Professional Art Printing Data </h1> 
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="holder">  
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+    </div>
+    </div>
     <button type="button" class="see-project"> See Project </button>
-  </form>
+    </form>
+    <form>
+      <div class="container">
+        <h1> Professional Art Printing Data </h1> 
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="holder">  
+          <ul>
+            <li>Html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+      </div>
+      </div>
+      <button type="button" class="see-project"> See Project </button>
+      </form>
+      <form>
+        <div class="container">
+          <h1> Professional Art Printing Data </h1> 
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
```

Comment:
- [ ] You did an amazing job but  following semantic HTML you are not allowed to use the h1 tag more than once. So please change it. 💪

Reply:
Thank you @Donard97  for constructive review;

On every line of code with <h1> Professional Art Printing Data </h1>  It is a repetitive card on which was duplicated on the design as referred to the design, that is why there is that repetition of <h3>

![1212](https://user-images.githubusercontent.com/50297713/153688965-f4b82e0f-a68b-4c79-b29a-3d401adb8163.PNG)


I have corrected other h1 that are duplicated, I submit my code now with all the changes needed.

Thank you very much.